code/modules/clothing/clothing.dm

```diff
@@ -477,6 +477,13 @@ BLIND     // can't see anything
 			. += mutable_appearance('icons/effects/item_damage.dmi', "damaged[blood_overlay_type]")
 		if(blood_DNA)
 			. += mutable_appearance('icons/effects/blood.dmi', "[blood_overlay_type]blood")
+		var/mob/living/carbon/human/M = loc

```

Comment:
There is a problem with that idea. It'll make the accessory overlap with things it shouldn't. For example, backpacks. This used the following: `accessory_overlay = mutable_appearance('icons/mob/accessories.dmi', "[accessory_color]",layer = ABOVE_MOB_LAYER)`
![layerproblem](https://user-images.githubusercontent.com/9963278/27174306-7d4db1a6-5170-11e7-9227-99a44a0d8cf6.PNG)